events/wc_primitive_events.txt

```diff
@@ -781,11 +785,153 @@ character_event = {
 	picture = GFX_evt_monk_water_deities
 	desc = EVTDESC_WCPRM_605
 	border = GFX_event_normal_frame_religion
-
+	
 	is_triggered_only = yes
-
+	
 	# Old fool!
 	option = {
 		name = EVTOPTA_WCPRM_605
 	}
 }
+
+character_event = {
+    id = WCPRM.500
+    picture = GFX_evt_kobold_statute
+    desc = EVTDESC_WCPRM_500
+    border = GFX_event_normal_frame_religion
+	
+    is_triggered_only = yes
+	
+	option = {
+	    name = EVTOPTA_WCPRM_500
+		piety = 1
+		wealth = -25
+		
+		random_list = {
+		    20 = {
+			    character_event = {
+				    id = WCPRM.501 days = 7
+				}
+			}
+		    30 = {
+			    character_event = {
+			        id = WCPRM.502 days = 7
+				}
+			}
+			50 = {
+			    character_event = {
+				    id = WCPRM.503 days = 7
+				}
+			}
+		}
+	}
+	
+	option = {
+		name = EVTOPTB_WCPRM_500
+	}
```

Comment:
The AI picks these options randomly.
https://ck2.paradoxwikis.com/Event_modding#AI_chance